src/lib/InstantSearch.ts

```diff
@@ -633,32 +639,27 @@ See ${createDocumentationLink({
         ? uiState(this.mainIndex.getWidgetUiState({}) as TUiState)
         : uiState;
 
-    const setIndexHelperState = (indexWidget: IndexWidget) => {
```

Comment:
this function is extracted to an utility, as once uiState became an argument, it was no longer needed to redeclare it on every call. This should in theory make changing state/searching a little faster (probably imperceptible)